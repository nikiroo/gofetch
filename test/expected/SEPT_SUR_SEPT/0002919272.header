0Un béluga aperçu dans la Tamise, à 50km de Londres	null/SEPT_SUR_SEPT/0002919272		70
i  [Espèces Menacées] 26-09-2018 13:15
i
i    Un   béluga,   cétacé  habituellement  observé  dans  l%27océan
i    Arctique,   a   été   aperçu   dans   la   Tamise,   provoquant
i    l%27inquiétude des défenseurs de ...
i
