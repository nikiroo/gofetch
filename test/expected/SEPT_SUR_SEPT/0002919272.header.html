<!DOCTYPE html>
<html>
<head>
  <meta http-equiv='content-type' content='text/html; charset=utf-8'>
  <meta name='viewport' content='width=device-width, initial-scale=1.0'>
  <style type='text/css'>
    body { margin: 1em 15%; }
  </style>
</head>
<body>
<div class='story-header'>
	<h2><a href='0002919272.html'>Un béluga aperçu dans la Tamise, à 50km de Londres</a></h2>
	<div class='details'>([Espèces Menacées] 26-09-2018 13:15)</div>
	<br/>
	<div class='content' style='text-align: justify'>
		Un béluga, cétacé habituellement observé dans l%27océan Arctique, a été aperçu dans la Tamise, provoquant l%27inquiétude des défenseurs de ...
	</div>
<hr/>
</div>
</body>
